src/components/icons/index.ts

```diff
@@ -73,6 +74,7 @@ export {
   CopyIcon,
   DashboardIcon,
   DocsIcon,
+  DownloadIcon,
```

Comment:
nit: start using lucide-react

Reply:
There's already a story filed to migrate to lucide-react. Will be done all together.